CONTRIBUTING.md

```diff
@@ -3,10 +3,12 @@ developed at [Jane Street][js] and is now maintained by Jane Street,
 [OCamlLabs][ocl] as well as several developers from the OCaml
 community.
 
-Contributions to Dune are welcome and should be submitted via GitHub
-pull requests against the `main` branch. Dune is distributed under
-the MIT license and contributors are required to sign their work in
-order to certify that they have the right to submit it under this
+Contributions to Dune are welcome and should be submitted via GitHub pull
+requests against the `main` branch. See [./doc/hacking.rst](./doc/hacking.rst)
+for a guide to getting started on the code base.
+
+Dune is distributed under the MIT license and contributors are required to sign
+their work in order to certify that they have the right to submit it under this
```

Comment:
Could you wrap these tighter?

Reply:
Hi. I'm not opposed to restructuring the lines, but these lines are currently all under 80 chars (which accords with the style guide indicated at line 80 -- by a neat coincidence -- in this same file ;) ).

What character limit would you prefer to enforce here?

Reply:
In that case that's fine, I was just thinking it was too long looking above and below.

Reply:
Ah, I hadn't noticed that difference with other lines in this file. Looks like they are wrapped to 70 cols. I've wrapped these accordingly, and it does look more uniform now. Thanks :) 6a6f65bada5fe48c20bedbd842f72e175924a4e2